### PR TITLE
chore: playground

### PR DESCRIPTION
**Pull request by geofmureithi**
This introduces a basic playground where one can write, compile and download contracts.

### CODE REVIEW

File: sample-contract/src/lib.rs.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+use soroban_sdk::{contractimpl, vec, Env, Symbol, Vec};
+
+pub struct Contract;
+    
+#[contractimpl]
+impl Contract {
+    pub fn hello(env: Env, to: Symbol) -> Vec<Symbol> {
+        vec![&env, Symbol::short("Hello"), to]
+    }
+}
```
Comment (mariopil) [inline sample-contract/src/lib.rs.example:11]: Missing new line

File: sample-contract/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[profile.release]
+opt-level = "z"
+overflow-checks = true
+debug = 0
+strip = "symbols"
+debug-assertions = false
+panic = "abort"
+codegen-units = 1
+lto = true
```
Comment (mariopil) [inline sample-contract/Cargo.toml:23]: Missing new line

File: sample-contract/.gitignore [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
+
+
+src/lib.rs
```
Comment (mariopil) [inline sample-contract/.gitignore:17]: Missing new line

File: playground/src/error.rs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            StatusCode::REQUEST_TIMEOUT,
+            "Request took too long".to_string(),
+        )
+    } else {
+        (
+            StatusCode::INTERNAL_SERVER_ERROR,
+            format!("Unhandled internal error: {}", err),
+        )
+    }
+}
```
Comment (mariopil) [inline playground/src/error.rs:54]: Missing new line

**Review by mariopil — APPROVED** [review]

**Review by eloylp — APPROVED** [review]
I like the changes ! Nitpick - it might be better to put this directly in the IDE repo (create if not exist) but we can also move it later. Leaving this decision for the author :)

**Pull request closed without merging**